File: src/test/java/no/nav/fo/veilarbregistrering/arbeidssoker/adapter/FormidlingsgruppeGatewayConfig.java
package no.nav.fo.veilarbregistrering.arbeidssoker.adapter;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class FormidlingsgruppeGatewayConfig {

    @Bean
    FormidlingsgruppeRestClient formidlingsgruppeRestClient() {
        return new FormidlingsgruppeRestClientMock();
    }
}

File: src/test/java/no/nav/fo/veilarbregistrering/arbeidssoker/adapter/FormidlingsgruppeGatewayConfig.kt
package no.nav.fo.veilarbregistrering.arbeidssoker.adapter

import no.nav.fo.veilarbregistrering.arbeidssoker.FormidlingsgruppeGateway
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class FormidlingsgruppeGatewayConfig {

    @Bean
    fun formidlingsgruppeRestClient(): FormidlingsgruppeRestClient {
        return FormidlingsgruppeRestClientMock()
    }

    @Bean
    fun formidlingsgruppeGateway(
        formidlingsgruppeRestClient: FormidlingsgruppeRestClient
    ): FormidlingsgruppeGateway {
        return FormidlingsgruppeGatewayImpl(formidlingsgruppeRestClient)
    }
}
